tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseTablespace.java

```diff
@@ -93,21 +94,13 @@ public void storageInit() throws IOException {
     hbaseConf.set(HConstants.ZOOKEEPER_CLIENT_PORT, splits[1]);
   }
 
-  @Override
-  public void setConfig(String name, String value) {
-  }
-
-  @Override
-  public void setConfigs(Map<String, String> configs) {
-  }
-
   public Configuration getHbaseConf() {
     return hbaseConf;
   }
 
   @Override
-  public long getTableVolume(URI uri) throws IOException {
-    return 0;
+  public long getTableVolume(URI uri) throws UnsupportedException {
+    throw new UnsupportedException();
```

Comment:
Yes, you are right. NotImplementedException should be thrown when we will implement later. 
I found a workaround to get the total volume of hbase table, that is using the HDFS command ```hadoop fs -du```. This will be inefficient, but may be very helpful for query planning. So, I think we need to implement this function someday. What do you think?